pkg/prowgen/prowgen_konflux.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 const (
-	KonfluxBranchPrefix = "sync-konflux/"
+	KonfluxBranchPrefix = "sync-konflux-"
```

Comment:
This change is because in Git we cannot have both branch 'foo' and branch 'foo/bar'